Gaijinizer patch/CommonEvents_770.txt

```diff
@@ -3,69 +3,93 @@
 \\N[1]よ……
 
 ヴァソパイアに負けてしまうとは、情けない…
-> CONTEXT: CommonEvents_770/9/Dialogue/enemy35 < UNTRANSLATED
+> CONTEXT: CommonEvents_770/9/Dialogue/enemy35
+\\N[1]...
 
+Losing to a vanpire is pathetic...#Ron
 > END STRING
 
 > BEGIN STRING
 ヴァソパイアは魔法中心の攻撃ではあるが
 万能属性も物理攻撃もこなす、厄介な相手だ……
 
 \\C[2]まずはダメージを極力減らす対策をした方がいい
-> CONTEXT: CommonEvents_770/16/Dialogue/enemy35 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_770/16/Dialogue/enemy35
+Though Vanpires tend to focus on magic for their
+attacks, they're also more than capable of using
+physical elemental attacks, making them quite a
+nuisance to fight.
```

Comment:
I'd say "physical elemental attacks" -> "physical attribute attacks", or possibly remove the middle word entirely.